### DISCUSSION TITLE
Gauge failure in devlopment version

### DISCUSSION THREAD

**Discussion by ianhaw** [flybywiresim/aircraft#6663] [Help] [answered]
Are you aware that the latest flybywire development version has total gauge failure with zero battery level in the A320neo!!

**Comment by aguther** [comment, marked as answer]
Well, you can imagine if that were true for all, we would get a lot of reports. Reading available documentation is always a clever idea:
https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/?h=reported#autopilot-fadec-electrical-systems-not-working-as-intended-utf8-issue
